Src/LinearSolvers/MLMG/AMReX_MLCGSolver.H

```diff
@@ -306,21 +306,17 @@ MLCGSolverT<MF>::solve_cg (MF& sol, const MF& rhs, RT eps_rel, RT eps_abs)
         }
         else
         {
-            RT beta = rho/rho_1;
+            const RT beta = rho/rho_1;
             MF::Xpay(p, beta, r, 0, 0, ncomp, nghost); // p = r + beta * p
         }
         Lp.apply(amrlev, mglev, q, p, MLLinOpT<MF>::BCMode::Homogeneous, MLLinOpT<MF>::StateMode::Correction);
 
-        RT alpha;
-        RT pw = dotxy(p,q);
-        if ( pw != RT(0.0))
-        {
-            alpha = rho/pw;
-        }
-        else
+        const RT pw = dotxy(p,q);
+        if ( pw == RT(0.0) )
         {
             ret = 1; break;
         }
+        const RT alpha = rho/pw;
```

Comment:
This restructuring of the divide by zero check allows `alpha` to be defined and declared in one line as a `const RT` instead of the current `RT` declaration.